Makefile

```diff
@@ -212,6 +209,11 @@ codecov: default
 	
 	+LD_LIBRARY_PATH="$$LD_LIBRARY_PATH:$$PWD/target/$(CARGO_TARGET)/$(CARGO_PROFILE_DIRECTORY)" $(CARGO) tarpaulin --all --verbose \
 		--exclude aziot-key-openssl-engine-shared \
+		--exclude openssl-build --exclude test-common \
+		--exclude-files identity/mock-dps-server/* \
```

Comment:
Why the distinction between `--exclude` for some directories and `--exclude-files` for others? Shouldn't `--exclude identity/mock-dps-server` work?

Reply:
`--exclude` takes a crate name, so it would be `--exclude mock-dps-server`, but yes it should be made consistent.

Reply:
If thats the case then for tpm/aziot-tpm-sys/azure-iot-hsm-c/deps and the aziot-tpm-sys test code I'll have to keep it as a wildcard to exclude all files in that folder?

Reply:
Probably. Try `--exclude aziot-tpm-sys` and see if it's smart enough to also exclude the C files, but I doubt it, since the object files are just registered for being linked in rather than part of the crate itself.

Reply:
Yup. It didn't exclude the C files . Updated the PR with a more consistent arrangement